resources/views/components/edit.blade.php

```diff
@@ -11,7 +11,7 @@
 @section('inputFields')
 
 @include ('partials.forms.edit.name', ['translated_name' => trans('admin/components/table.title')])
-@include ('partials.forms.edit.category-select', ['translated_name' => trans('general.category'), 'fieldname' => 'category_id','category_type' => 'component'])
+@include ('partials.forms.edit.category-select', ['translated_name' => trans('general.category'), 'fieldname' => 'category_id','category_type' => $category_type])
```

Comment:
Where is `$category_type` coming from in this blade? (I can't tell from this diff.) The category type should still be components for components, consumable for consumables, etc, no? What do we gain by making this a variable?

Reply:
From each Controller, I think my objective was to made it more flexible, because with this change, theoretically we can use the same view for edit every category. But ended with that variable in every edit template. I want to discuss that change, but time passed on and I'm unable to do so. So it ended like this.